packages/modeling/src/geometries/geom3/fromPointsConvex.test.js

```diff
@@ -0,0 +1,25 @@
+const test = require('ava')
+
+const { fromPointsConvex } = require('./index')
+
+test('fromPointsConvex (listofpoints)', (t) => {
+  let out = []
+  for(x=-9;x<=9;++x)
+    for(y=-9;y<=9;++y)
+     for(z=-9;z<=9;++z)
+        if (x*x+y*y+z*z <= 96)
+          out.push([x,y,z])
+
+  let obs = fromPointsConvex(out)
+  t.is(obs.polygons.length, 170)
+  t.true(obs.polygons.every((f) => ([3,4,8,9].indexOf(f.vertices.length) !== -1)))
```

Comment:
Probably, the only other check would be..

`t.true(geom3.isValid(obs))`

very simple check for completeness.

Reply:
Hi, I did the uniquePoints changes.
There is no geom3.isValid(), so I added validate(obs) instead.
```
hermann@j4105:~/OpenJSCAD.org/packages/modeling/src/geometries/geom3$ /home/hermann/OpenJSCAD.org/node_modules/.bin/ava "*.test.js" --verbose --timeout 2m

  ✔ clone › clone: Creates a clone on an empty geom3
  ✔ clone › clone: Creates a clone of a populated geom3
  ✔ applyTransforms › applyTransforms: Updates a geom3 with transformed polygons
  ✔ create › create: Creates an empty geom3
  ✔ create › create: Creates a populated geom3
  ✔ fromPoints › fromPoints: Creates a populated geom3
  ✔ fromPoints › fromPoints: throws for improper points
  ✔ invert › invert: Creates a invert on an empty geom3
  ✔ invert › invert: Creates a invert of a populated geom3
  ✔ fromToCompactBinary › toCompactBinary: converts geom3 (default)
  ✔ isA › isA: identifies created geom3
  ✔ isA › isA: identifies non geom3
  ✔ fromToCompactBinary › toCompactBinary: converts geom3 into a compact form
  ✔ fromToCompactBinary › fromCompactBinary: convert a compact form into a geom3
  ✔ fromPointsConvex › fromPointsConvex (uniquePoints) (130ms)
  ✔ toPoints › toPoints: Creates an array of points from a populated geom3
  ✔ transform › transform: Adjusts the transforms of a populated geom3
  ─

  17 tests passed
hermann@j4105:~/OpenJSCAD.org/packages/modeling/src/geometries/geom3$ 
```